src/Augurk/Web.config

```diff
@@ -25,7 +25,9 @@
       <remove name="OPTIONSVerbHandler" />
       <remove name="TRACEVerbHandler" />
       <add name="ExtensionlessUrlHandler-Integrated-4.0" path="*." verb="*" type="System.Web.Handlers.TransferRequestHandler" preCondition="integratedMode,runtimeVersionv4.0" />
+      <add name="aspNetCore" path="*" verb="*" modules="AspNetCoreModule" resourceType="Unspecified" />
     </handlers>
+    <aspNetCore processPath="%LAUNCHER_PATH%" arguments="%LAUNCHER_ARGS%" stdoutLogEnabled="false" />
   </system.webServer>
   <runtime>
```

Comment:
I think we can remove the bindingRedirects now, since they are no longer necessary.

Reply:
Agreed

Reply:
I have removed pretty much everything from the Web.config in a later pull request, since all we really need is this new line which is automatically generated on build anyway.